specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json

```diff
@@ -0,0 +1,171 @@
+            "identityClientId": ""
+          }
+        },
+        "hbiWorkspace": false,
+        "sharedPrivateLinkResources": [
+          {
+            "name": "testcosmosdbresource",
+            "properties": {
+              "privateLinkResourceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.DocumentDB/databaseAccounts/testcosmosdbresource/privateLinkResources/Sql",
+              "groupId": "Sql",
+              "requestMessage": "Please approve",
+              "status": "Approved"
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.MachineLearningServices/workspaces/testworkspace",
+        "name": "testworkspace",
+        "type": "Microsoft.MachineLearningServices/workspaces",
+        "location": "eastus2euap",
+        "sku": {
+          "name": "Basic",
+          "tier": "Basic"
+        },
+        "properties": {
+          "containerRegistry": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.ContainerRegistry/registries/testRegistry",
+          "keyVault": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.KeyVault/vaults/testkv",
+          "applicationInsights": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/microsoft.insights/components/testinsights",
+          "storageAccount": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/accountcrud-1234/providers/Microsoft.Storage/storageAccounts/testStorageAccount",
+          "discoveryUrl": "http://example.com",
+          "creationTime": "2017-03-01T23:14:37.0707808Z",
+          "friendlyName": "HelloName",
+          "description": "test description",
+          "encryption": {
+            "status": "Enabled",
+            "keyVaultProperties": {
+              "keyVaultArmId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.KeyVault/vaults/testkv",
+              "keyIdentifier": "https://testkv.vault.azure.net/keys/testkey/aabbccddee112233445566778899aabb",
+              "identityClientId": ""
+            }
+          },
+          "hbiWorkspace": false,
+          "sharedPrivateLinkResources": [
+            {
+              "name": "testcosmosdbresource",
+              "properties": {
+                "privateLinkResourceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.DocumentDB/databaseAccounts/testcosmosdbresource/privateLinkResources/Sql",
+                "groupId": "Sql",
+                "requestMessage": "Please approve",
+                "status": "Approved"
+              }
+            }
+          ]
+        }
+      }
+    },
+    "201": {
```

Comment:
also, 201 shouldn't return response body

Reply:
I think 201 should still return body but 202 shoudn't